### PR TITLE
Squad Leader Vendor/Essential Update

### PR DESCRIPTION
**Pull request by Surrealaser**
-SL essential kit loses flamethrower and ziptie boxes (wtf) and gains a supply beacon, a health HUD for helping assess/coordinate his team, and a field utility pouch. This pouch contains a motion detector, whistle, station bounced radio, tactical binoculars and map. These replace the items that used to be issued loose leaf in the kit.

-Full basic construction pouches (some sandbags/barbed wire and entrenching tool) added to SL vendor.

-Medikit pouch added to SL vendor (to give him a place to stash a medikit if he pointbuys one).

-SL can now point buy an entrenching tool for his sandbag purchases if any.

-Zipties boxes now cost 5 points at the vendor.

-Cost reductions:
Flamethrower tank 6 > 4 (brings it in line with HPR ammo cost)
Orbital beacon 25 > 15 (no idea why this was so insanely overcosted).
Supply beacon 15 > 10
25x Sandbags 15 > 10
Plastique explosive 5 > 3
Incendiary grenade 8 > 4

-Removed the tactical pouch from essentials and added its content to essentials.

-Made it so general pouches don't have special weight class exceptions for ammo.

-Reorganized and added some utility items to the SL vendor, including the 'cloak' smoke grenade that's apparently supposed to work vs Xenos.

-SL can purchase frag grenades with point buy; don't see why he can buy incendiaries but not these.

-Reduced the size of the motion detector by one step, making it able to fit in more containers.

### CODE REVIEW

File: code/game/objects/items/storage/pouch.dm [hunk 1]
```diff
@@ -372,10 +372,37 @@
 	storage_slots = 2
 	icon_state = "radio"
 	draw_mode = 1
-	desc = "It can contain two handheld radios."
+	desc = "It can contain two handheld radios"
```
Comment (Rohesie) [inline code/game/objects/items/storage/pouch.dm]: Why did you remove the full stop?

File: code/game/objects/items/storage/pouch.dm [hunk 2]
```diff
@@ -372,10 +372,37 @@
...
 	can_hold = list("/obj/item/device/radio")
 
 
+/obj/item/storage/pouch/field_pouch
```
Comment (Rohesie) [inline code/game/objects/items/storage/pouch.dm]: All of these fit in the large general pouch and webbing.

Comment (Rohesie) [reply code/game/objects/items/storage/pouch.dm]: You can make it fit there then, or alter its size.

Comment (Surrealaser) [reply code/game/objects/items/storage/pouch.dm]: Or we can add a pouch to essentials that people would be more inclined to take and use as it holds and comes preloaded with important and useful gear, and is part of that crucial tone setting kit, thus encouraging people to actually do the job they're intended to instead of taking a flamer/HPR and being a grunt +1.

**Comment by Surrealaser** [comment]
@Qustinnus Hey, could you approve this for test merge so we can at least try these changes?

File: code/game/objects/items/storage/pouch.dm [hunk 3]
```diff
@@ -376,6 +376,33 @@
...
+					"/obj/item/device/megaphone",
+					"/obj/item/device/flashlight",
+					"/obj/item/device/whistle",
+					"/obj/item/device/binoculars",
+					"/obj/item/map/current_map",
+					"/obj/item/map/squad_beacon",
+					)
+
+/obj/item/storage/pouch/field_pouch/full/New()
+	..()
```
Comment (Qustinnus) [inline code/game/objects/items/storage/pouch.dm]: .=..()
